charts/kcp-sharded/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
```

Comment:
Can we maybe rename this chart to `shard` (to be in line with the other new charts) and then maybe have a `kcp-sharded` umbrella chart that pulls in all of the "building block" charts? The naming seems inconsistent that way.